packages/zipkin/src/jsonEncoder.js

```diff
@@ -145,8 +145,15 @@ function encodeV2(span) {
   if (span.annotations.length > 0) {
     copy.annotations = span.annotations;
   }
-  if (Object.keys(span.tags).length > 0) {
-    copy.tags = span.tags;
+  const tagKeys = Object.keys(span.tags);
+  if (tagKeys.length > 0) {
+    copy.tags = copy.tags || {};
+    for (var k of tagKeys) {
+      const val = span.tags[k];
+      if (val !== null && val !== undefined) {
```

Comment:
I think it would be better to enforce this when creating the span tags as opposed to going to json (ex would be same issue in proto)